### ISSUE TITLE
Plans for `v2.0.0`

### ISSUE THREAD

**Issue by Ultra-Instinct-05** [Sublime-Instincts/BetterTwig#18]
Now that Sublime Text 4 has had a few stable releases, here's the rough draft of the steps that needs to be followed to cut out a `v2.0.0` release & move forward with refactoring the syntax to support inheritance.

- [x] Cut one final release for ST3 with the tag `st3-v1.0.7` with no new changes.
- [x] Make a PR to `package_control_channel` with the following changes for `Twig`
```json
"releases": [
    {
        "sublime_text": "<4107",
        "tags": "st3-"
    },
    {
        "sublime_text": ">=4107",
        "tags": true
    }
]
```
- [x] Cut out a `v2.0.0` release implementing Inheritance (There should be no user facing changes with this however. Inheritance will simply cut down the regex & syntax cache on disk and improve performance in general).

Move forward with further improvements & enhancements. The `v2.0.0` will not be a breaking change in any sense even though the major version is incremented (which is only done because we are moving forward with ST4 and removing any support for ST3)

**Comment by Ultra-Instinct-05** [comment]
The 2nd task is being tracked over at https://github.com/wbond/package_control_channel/pull/8413

**Comment by Ultra-Instinct-05** [comment]
I guess since it is `v2.0.0`, It's better to split the syntax into 2 parts `Twig.sublime-syntax` & `HTML (Twig).sublime-syntax`. `HTML (Twig).sublime-syntax` will extend `HTML.sublime-syntax` & `Twig.sublime-syntax` to provide what the current `Twig.sublime-syntax` provides i.e. HTML + Twig highlighting, while the new `Twig.sublime-syntax` will be a bare bones syntax that will just implement the Twig syntax without any HTML stuff.

The only issues this will have is:

1. Any references to `Twig.sublime-syntax` must change to `HTML (Twig).sublime-syntax` (If you have used it anywhere in key bindings, settings etc.)
2. If you have any open Twig files, then if the package upgrades to `v2.0.0`, you will loose highlighting for HTML stuff. This can be remedied by opening any file you want both HTML + Twig and following https://github.com/Sublime-Instincts/BetterTwig#how-to-use-this-package- but instead of `Twig`, you'd select `HTML (Twig)`

**Comment by Ultra-Instinct-05** [comment]
All the tasks are now completed. I am planning on releasing `v2.0.0` sometime before the end of December.

**Comment by Ultra-Instinct-05** [comment]
Addressed with `v2.0.0` being released.
